torch/jit/_script.py

```diff
@@ -53,6 +53,11 @@ def Attribute(value, type):  # type: ignore
         return value
 
 
+# Gets a function from the name of a method on a type
+def _get_function_from_type(cls, name):
```

Comment:
nit: not sure if this function is really necessary? Could you just have the two getattr calls below?

Reply:
I did try that but the CI failed. So I restored it in https://github.com/pytorch/pytorch/commit/d86e3f45850497eaad77ba62bc5da6ab57800a2b

See https://github.com/pytorch/pytorch/pull/49785#issuecomment-752850244